docker/allensdk/Dockerfile

```diff
@@ -1,11 +1,29 @@
-# DOCKER-VERSION 1.11.1
+# DOCKER-VERSION 2.2.0.5
 #
 # docker build --tag alleninstitute/allensdk .
 # docker run -it alleninstitute/allensdk /bin/bash
 #
-FROM alleninstitute/anaconda_neuron_1370
+FROM continuumio/anaconda3
 
-MAINTAINER Tim Fliss <timf@alleninstitute.org>
+WORKDIR /root
 
-COPY . ./allensdk/
-RUN  pip install ./allensdk
\ No newline at end of file
+# Add AllenSDK and NEURON libraries
+RUN pip install -q allensdk neuron
```

Comment:
I would recommend removing the `-q` option. The output could end up being very helpful with debugging build issues.

Reply:
Should be fixed.